crates/bevy_pbr/src/light.rs

```diff
@@ -712,8 +712,13 @@ pub(crate) fn assign_lights_to_clusters(
     lights_query: Query<(Entity, &GlobalTransform, &PointLight, &Visibility)>,
     mut lights: Local<Vec<PointLightAssignmentData>>,
     mut max_point_lights_warning_emitted: Local<bool>,
-    render_device: Res<RenderDevice>,
+    render_device: Option<Res<RenderDevice>>,
 ) {
+    let render_device = match render_device {
+        Some(render_device) => render_device,
+        None => return,
```

Comment:
I wish there were a nicer syntax for doing this, like a ? operator that just returns if none else unwraps it. But I’m not aware of any.

Reply:
[Let else](rust-lang/rfcs#3137) will help with this slightly.